### PR TITLE
Remove resolvconf from Ubuntu 18

### PR DESCRIPTION
**Pull request by jmatsushita**
Aiming to simplify the dns setup on concourse workers which experience DNS flakyness.

### CODE REVIEW

**Review by arianvp — APPROVED** [review]
LGTM

**Comment by arianvp** [comment]
One note. We're removing `resolvconf`; not `resolv.conf`. Might want to change the commit message to avoid confusion :)

File: tasks/main.yml [hunk 1]
```diff
@@ -103,6 +103,11 @@
     backup=yes
   notify:
     - regenerate resolv.conf
+  when: ansible_distribution_version != '18.04'
+
+- name: ensure resolv.conf is /run/systemd/resolve/resolv.conf
+  file: src=/run/systemd/resolve/resolv.conf dest=/etc/resolv.conf state=link
```
Comment (arianvp) [inline tasks/main.yml:109]: I'd prefer if we use the new ansible syntax for new changes where possible. The old syntax is going to be unsupported at some point.

```yaml
file:
  src:
  dest:
  state:
```

Comment (jmatsushita) [reply tasks/main.yml]: 👍

**Comment by jmatsushita** [comment]
> Might want to change the commit message to avoid confusion :)

👍

**Pull request merged**